cg2019_paper.tex

```diff
@@ -199,7 +200,7 @@ \section{State of the Art}
 itself offers a baseline controller, but that is also configurable and
 you can choose to wire it in some other way
 \cite{cussat2016dangerousness}. While this might be effective for some
-specific purposes like the one used in the paper, evaluate the
+specific purposes like the one used in the paper,\textcolor{red}{to} evaluate the
```

Comment:
Eliminate comma here?

Reply:
That depends. Is that a clarification about "the one used in the paper"? I read the the sentence as "I am remanding you what it was just in case" so the comma would be alright.